### PR TITLE
Add universal viewer via arclight's Viewer Class extension point

### PR DESCRIPTION
**Pull request by hackmastera**
moves #35 to demo column

### CODE REVIEW

**Comment by hackmastera** [comment]
Okay this is ready for review.

Since I changed the code to not actually hit solr, I removed the commit that does the solr cleaning. I also created https://github.com/pulibrary/pulfalight/issues/183

Screenshots for this PR are as posted on #161

File: app/views/viewers/_universal_viewer.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="uv__overlay" onClick="style.pointerEvents='none'">
+  <iframe src="<%= viewer.url %>" allowfullscreen="true"></iframe>
+  <%= link_to image_tag("iiif.png", class: "logo", alt: "IIIF Drag and Drop"), viewer.url %>
```
Comment (jrgriffiniii) [inline app/views/viewers/_universal_viewer.html.erb]: 👍 thank you for including this.

File: config/initializers/plantain_config.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Plantain
+  def config
+    @config ||= config_yaml.with_indifferent_access
+  end
+
+  private
+
+    def config_yaml
+      # This invocation will change in ruby 2.6 to
```
Comment (jrgriffiniii) [inline config/initializers/plantain_config.rb]: 👍

**Comment by jrgriffiniii** [comment]
https://pulfalight-staging.princeton.edu/catalog/C0385C0385_c052 should work, but the Loris IIIF Image Server endpoint is responding with a 502 (this is unrelated). However, the Manifest from Figgy is successfully retrieved.

File: config/initializers/plantain_config.rb (hunk 2 above)
Comment (hackmastera) [reply config/initializers/plantain_config.rb]: Oh hey we're on ruby 2.6 now. I should update this.

**Pull request merged**